### ISSUE TITLE
Calendar button titles and "Go to today" focus

### ISSUE THREAD

**Issue by cjsheets** [OfficeDev/office-ui-fabric-react#9194]
This issue relates to a subset of [Calendar A11y Pass #8349](https://github.com/OfficeDev/office-ui-fabric-react/issues/8349) (specifically "Bug1" and "Bug2")

## Codepen demonstrating the issue
[Standard Calendar](https://codepen.io/anon/pen/zQRmRw?&editable=true)

## Actual behavior:
* Icon buttons for next/prev month should include tooltips for users who don't use a screenreader
* Focus is lost when GoToToday is pressed

## Expected behavior:
* Tooltip is present on icons without other labels
* Focus returns to current day when GoToToday is pressed

## Documentation describing expected behavior

The desired behavior was flagged by A11Y auditors.
 [name computation](https://www.w3.org/TR/wai-aria-1.1/#namecalculation) or purpose of [aria-label](https://www.w3.org/TR/wai-aria-1.1/#aria-label).

**Comment by micahgodbolt** [comment]
@lorejoh12 do you know if you've looked into tooltips for those buttons? Is there a reason they aren't there? Do you have any feedback on the "GoToToday" functionality? Is it working within the Date-Time version?

Lots of questions, right?

**Comment by kenotron** [comment]
I think this is a feature that has not been implemented yet. So, probably not yet a bug? :)

**Comment by cjsheets** [comment]
The tooltip is the new feature right? Would it help if I broke the tooltip and GoToToday changes into 2 PRs so we can address them seperatly?

Currently, when you click "GoToToday" focus remains on the now-disabled GoToToday button which prevents Enter from working and prevents ESC from closing callouts (in the case of the Datepicker). This was flagged as a high impact AX issue for us.

If I need to temporally work around the tooltip I can but the onClick behavior of GoToToday is more difficult. FWIW, I validated the calendars I could find locally and on the [PR site](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/9195/merge/fabric-website-resources/dist/index.html#/examples/datepicker).
